src/example/org/deidentifier/arx/examples/Example59.java

```diff
@@ -1,44 +1,86 @@
 package org.deidentifier.arx.examples;
 
-import org.deidentifier.arx.framework.data.DataColumn;
-import org.deidentifier.arx.framework.data.DataMatrix;
-import org.deidentifier.arx.framework.data.Dictionary;
-import org.deidentifier.arx.masking.DataMaskingFunction.PermutationFunctionColumns;
+import java.io.IOException;
+import java.util.Arrays;
+import java.util.Iterator;
 
-public class Example59 {
-	
-	public static void main (String[] args) {
-		int rows = 1000;
-		int columns = 2;
-		
-		DataMatrix data = new DataMatrix(rows, columns);
-		String[] header = new String[]{"column1","column2"};
-		int column = 0;
-		Dictionary dictionary = new Dictionary(rows);
+import org.deidentifier.arx.ARXAnonymizer;
+import org.deidentifier.arx.ARXConfiguration;
+import org.deidentifier.arx.ARXResult;
+import org.deidentifier.arx.AttributeType;
+import org.deidentifier.arx.Data;
+import org.deidentifier.arx.AttributeType.Hierarchy;
+import org.deidentifier.arx.AttributeType.MaskingFunction;
+import org.deidentifier.arx.AttributeType.Hierarchy.DefaultHierarchy;
+import org.deidentifier.arx.Data.DefaultData;
+import org.deidentifier.arx.criteria.KAnonymity;
 
-		int[] tuple = new int[columns];
-		
-		for (int i=0;i<rows;i++) {
-			
-			for(int j=0;j<columns;j++) {
-				tuple[j] = dictionary.register(j, Integer.toString(j+1) + "_ColumnStringTest_" + Integer.toString(i+1)+"row");
-			}
-			data.setRow(i, tuple);
-		}
-		
-		
-		dictionary.finalizeAll();
-		DataColumn c = new DataColumn(data, header, column, dictionary);
-		
-		PermutationFunctionColumns pfc = new PermutationFunctionColumns(true);
-		for (int i=0; i<c.getNumRows();i++) {
-			System.out.println(c.get(i));
-		}
-		System.out.println();
-		pfc.apply(c);
+/**
+ * This class implements an example of permutation function on columns
+ * 
+ * @author giupardeb
```

Comment:
If you want you can add your full name, but we are also fine with a pseudonym.

Reply:
in this moment I prefer to use the pseudonym, thanks!